subsys/bluetooth/controller/Kconfig

```diff
@@ -6,6 +6,24 @@
 
 menu "Nordic BLE controller"
 
+# BT_HCI_TX_STACK_SIZE is declared in Zephyr and also here for a second time,
+# for set the valid default size of the HCI Tx stack.
+config BT_HCI_TX_STACK_SIZE
```

Comment:
Should we set a default RX_STACK_SIZE as well?

Reply:
Hard to say because I do not know about any special requirments for RX stack for Nordic controller.

Reply:
The TX thread will call the BLE controllers data_put and command_put. While the RX thread will call the BLE controllers data_get and event_get, so I think it makes sense to set an RX thread stack size.

Reply:
Ok. I added configuration for the RX stack but It is the same like in upstream because I do not know if we need other default values for it